### PR TITLE
[Doc] Update contributing doc with cavaet 

### LINKED ISSUE #117: [Docs] Update SMP Contributing Doc
Add cavaet to contributing doc
CC/ @leecalcote 
---
**Contributor Resources**
- [Service Mesh Performance documentation source](https://github.com/layer5io/service-mesh-performance/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/service-mesh-performance/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

### PR DESCRIPTION
**Pull request by PluckyPrecious**
Signed-off-by: PluckyPrecious <prekel27@gmail.com>

**Description**
Added Jekyll cavaet and some fixes.
This PR fixes #117 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by JubayerJoy — COMMENTED** [review]
LGTM

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -44,9 +44,11 @@ Or you may configure your IDE, for example, Visual Studio Code to automatically
...
 
-### Set up your development environment
+## <a name="setting up environment">Set up Your Development Environment</a>
+*The Service Mesh Performance site is built using Jekyll - a simple static site generator! You can learn more about Jekyll and setting up your development environment in the [Jekyll Docs](https://jekyllrb.com/docs/).*
+* First [install Ruby](https://jekyllrb.com/docs/installation/), then install Jekyll and Bundler.
 
-Note: Windows users can run Jekyll by following the [Windows Installation Guide](https://jekyllrb.com/docs/installation/windows/), for Jekyll. This includes installing the Ruby Version Manager [RVM](https://rvm.io) - a command-line tool that allows you to easily install, manage, and work with multiple ruby environments on your local machine. 
+**Please Note:** Windows users can run Jekyll by following the [Windows Installation Guide](https://jekyllrb.com/docs/installation/windows/), for Jekyll. This includes installing the Ruby Version Manager [RVM](https://rvm.io) - a command-line tool that allows you to easily install, manage, and work with multiple ruby environments on your local machine.
```
Comment (kushthedude) [inline CONTRIBUTING.md:51]:
```suggestion
**Note:** Windows users can run Jekyll by following the [Windows Installation Guide](https://jekyllrb.com/docs/installation/windows/), for Jekyll. This includes installing the Ruby Version Manager [RVM](https://rvm.io) - a command-line tool that allows you to easily install, manage, and work with multiple ruby environments on your local machine. 
```

It is a note specifically.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -66,6 +68,7 @@ Please contribute! All projects under Layer5 use GitHub Pages to host its docume
 `vi <specific page>.md`
 1. Run site locally to preview changes.
 `make site`
+* **Please Note:** *From the Makefile, this command is actually running `$ bundle exec jekyll serve --drafts --livereload`. There are two Jekyll configuration, `jekyll serve` for developing locally and `jekyll build` when deploying for production based on differences of the handling of trailing slashes between Jekyll and GitHub Pages.*
```
Comment (kushthedude) [inline CONTRIBUTING.md:71]:
```suggestion
* **Note:** *From the Makefile, this command is actually running `$ bundle exec jekyll serve --drafts --livereload`. There are two Jekyll configuration, `jekyll serve` for developing locally and `jekyll build` when deploying for production based on differences of the handling of trailing slashes between Jekyll and GitHub Pages.*
```

Same as above

**Review by kushthedude — COMMENTED** [review]
Everything looks good, just few nit

**Review by leecalcote — APPROVED** [review]
LGTM

**Pull request merged**